proteomics.py

```diff
@@ -172,110 +184,116 @@ class IdXML(ProteomicsXml):
     blurb = "OpenMS identification file"
     root = "IdXML"
 
+
 class TandemXML(ProteomicsXml):
     file_ext = "tandem"
     blurb = "X!Tandem search results file"
     root = "bioml"
 
-class Mgf( Text ):
+
+class Mgf(Text):
     """Mascot Generic Format data"""
     file_ext = "mgf"
 
-    def set_peek( self, dataset, is_multi_byte=False ):
+    def set_peek(self, dataset, is_multi_byte=False):
         """Set the peek and blurb text"""
         if not dataset.dataset.purged:
-            dataset.peek = data.get_file_peek( dataset.file_name, is_multi_byte=is_multi_byte )
+            dataset.peek = data.get_file_peek(dataset.file_name, is_multi_byte=is_multi_byte)
             dataset.blurb = 'mgf Mascot Generic Format'
         else:
             dataset.peek = 'file does not exist'
             dataset.blurb = 'file purged from disk'
 
-    def sniff( self, filename ):
+    def sniff(self, filename):
         mgf_begin_ions = "BEGIN IONS"
-        max_lines=100
+        max_lines = 100
 
-        for i, line in enumerate( file( filename ) ):
-            line = line.rstrip( '\n\r' )
```

Comment:
I assume this should be a line break? `\n\r`
I removed this, because `strip()` removes all white space.